itest/tests/preferences.test.js

```diff
@@ -0,0 +1,30 @@
+/* @flow */
+import path from "path"
+
+import {
+  defaultModalButton,
+  jsonTypeConfigInput,
+  viewerResults
+} from "../../src/js/test/locators"
+import {stdTest} from "../lib/jest"
+import {testDataDir} from "../lib/env"
+import createTestBrim from "../lib/createTestBrim"
+
+const config = path.join(testDataDir(), "custom-schema.json")
+
+describe("Preferences Modal Tests", () => {
+  const brim = createTestBrim("preferences.test")
+
+  stdTest("setting a custom json type config", async (done) => {
+    brim.clickAppMenuItem("preferences")
+    await brim.setValue(jsonTypeConfigInput, config)
+    await brim.click(defaultModalButton)
+    await brim.ingest("custom-sample.ndjson")
+    await brim.search(
+      "_path=conn proto=tcp | cut ts, src_ip, src_port, dst_ip, dst_port, proto | sort ts"
+    )
+
+    expect(await brim.getText(viewerResults)).toMatchSnapshot()
```

Comment:
I wonder if you're getting "End of results" as shown above because you're not using the app step `searchDisplay()`?

Reply:
I looked at the `searchDisplay()` code and can't see any reason why it would exclude "End of results"...

Reply:
(Answer: different locators)